src/App.js

```diff
@@ -86,6 +87,7 @@ function App() {
                   </UserDataProvider>
                 }
               />
+              <Route path="in-progress" element={<InProgressPage />} />
```

Comment:
We can add navigate to that page to some of buttons/fake buttons right now, for example here:
![image](https://user-images.githubusercontent.com/92270179/152963976-88993bb0-3051-4292-8d62-5225b523e9e5.png)